cmd/dp-import-tracker/main.go

```diff
@@ -38,10 +38,18 @@ type inputFileAvailable struct {
 	InstanceID string `avro:"instance_id"`
 }
 
+type insertResult int
+
+const (
+	AllGood  insertResult = 0
```

Comment:
not used?

Reply:
I wanted an explicit zero-value, so that closing the channel was seen as `AllGood` (as I understand it, it would return zero if closed) even if I don't actually check for it.

Reply:
👍